nested/dialog-window.html

```diff
@@ -60,10 +61,12 @@ <h5>Are you sure to delete the screen shot</h3>
 
 		const window = WebviewWindow.getByLabel('main')
 		const dialogWindow = getCurrent()
+		const dialogWindow2 = WebviewWindow.getByLabel('dialog'); // reference to the WebviewWindow with the label 'dialog' to set decoration to false again.
```

Comment:
why did u need to created a second variable , when we already had refercences for the dialog window